### ISSUE TITLE
ISSUE#3 - Parse the dcs.log file

### ISSUE THREAD

**Issue by Shredmetal** [Shredmetal/olympus_discord_bot_python#13]
We're looking for a whole bunch of shit in there

**Comment by Shredmetal** [comment]
ok <@450395731359170590> in dcs.log you should see
```
2024-07-18 14:36:31.951 INFO    Olympus.HOOKS.LUA (Main): olympus.dll loaded successfully
2024-07-18 14:36:31.951 INFO    Olympus.HOOKS.LUA (Main): Olympus v1.0.3.2f43a84 C++ module callbacks registered correctly.
```
id regex the olympus version number just look for something there, but those two lines means its likely installed correctly

```
2024-07-18 14:36:31.949 INFO    Olympus.HOOKS.LUA (Main): Executing OlympusHook.lua
2024-07-18 14:36:31.949 INFO    Olympus.HOOKS.LUA (Main): Loading olympus.dll from [C:\Users\DCS\Saved Games\Borderzone1\Mods\Services\Olympus\bin\]
2024-07-18 14:36:31.951 INFO    Olympus.dll (Main): Instance location retrieved successfully
2024-07-18 14:36:31.951 INFO    Olympus.dll (Main): Loading .dlls from C:\Users\DCS\Saved Games\Borderzone1\Mods\Services\Olympus\bin\
```
this is what preceeds it, so if you see any of this but neither of the 2 lines from that bfirst bit then something is busted and its not loading, so basically if Olympus.HOOKS and/or Olympus.dll appear at all but both those lines above dont appear then something is borked with the install

if none of this appears then they installed to the wrong location

**Comment by Shredmetal** [comment]
resolved with PR#19
